tests/api.c

```diff
@@ -56351,7 +56351,7 @@ static int test_wolfSSL_RSA_verify(void)
     AssertTrue((fp != XBADFILE));
     AssertIntGE(XFSEEK(fp, 0, XSEEK_END), 0);
     sz = XFTELL(fp);
-    XREWIND(fp);
+    AssertIntGE(XFSEEK(fp, 0, XSEEK_SET), 0);
```

Comment:
AssertIntEQ?

Reply:
whoops, good catch -- will check for others like it.